src/main/java/org/openstreetmap/atlas/tags/FerryTag.java

```diff
@@ -0,0 +1,47 @@
+package org.openstreetmap.atlas.tags;
+
+import java.util.EnumSet;
+
+import org.openstreetmap.atlas.tags.annotations.Tag;
+import org.openstreetmap.atlas.tags.annotations.TagKey;
+
+/**
+ * OSM Ferry Tag
+ *
+ * @author isabellehillberg
+ */
+@Tag(taginfo = "https://taginfo.openstreetmap.org/keys/ferry#values")
```

Comment:
Nit: Can you link the OSM wiki please:

`osm = "https://wiki.openstreetmap.org/wiki/Key:ferry"`

Reply:
The key is not documented. The route=ferry tag is the official way to tag ferries in OSM, but the ferry tag is used by the community in some cases.

Reply:
It is - the wiki above! It's pretty brief but at least gives users a reference point. I'd add it for completeness.

Reply:
Got it!